ui/wobbly.js

```diff
@@ -307,18 +307,34 @@ function enableWobblyFx(wm) {
         SETTINGS_HANDLER = Settings.connect('changed::wobbly-effect', actionWobblyEffectSetting);
         actionWobblyEffectSetting(Settings, 'wobbly-effect');
     });
+
+    return GLib.SOURCE_REMOVE;
```

Comment:
I guess this is no longer needed with latest changes?

Reply:
Oops, yes.